dali/operators/decoder/image_decoder.cc

```diff
@@ -24,79 +24,114 @@ DALI_SCHEMA(ImageDecoderAttr)
-requested size per thread. If chosen correctly, no more allocations will occur during the pipeline
-execution. One way to find the ideal value is to do a full run over the dataset with the argument
-``memory_stats`` set to True and then copy the "biggest" allocation value printed in the statistics.)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+The padding for nvJPEG’s device memory allocations in bytes. This parameter helps you avoid
+reallocation in nvJPEG when a larger image is encountered, and the internal buffer needs to be
+reallocated to decode the image.
+
+If a value greater than 0 is provided, the operator preallocates one device buffer of the
+requested size per thread. If the value is correctly selected, no additional allocations
+will occur during the pipeline execution. One way to find the ideal value is to do a complete
+run over the dataset with the ``memory_stats`` argument set to True and then copy the largest
+allocation value that was printed in the statistics.)code",
       16*1024*1024)
   .AddOptionalArg("host_memory_padding",
-      R"code(**`mixed` backend only** Padding for nvJPEG's host memory allocations in bytes.
-This parameter helps to avoid reallocation in nvJPEG whenever a bigger image
-is encountered and internal buffer needs to be reallocated to decode it.
-
-If a value bigger than 0 is provided, the operator will pre-allocate two (because of double-buffering)
-host pinned buffers of the requested size per thread. If chosen correctly, no more allocations will occur
-during the pipeline execution. One way to find the ideal value is to do a full run over the dataset with the
-argument ``memory_stats`` set to True and then copy the "biggest" allocation value printed in the statistics.)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+The padding for nvJPEG’s host memory allocations in bytes. This parameter helps you prevent
+the reallocation in nvJPEG when a larger image is encountered, and the internal buffer needs
+to be reallocated to decode the image.
+
+If you provide a value greater than 0, the operator preallocates two (because of double-buffering)
+host-pinned buffers of the requested size per thread. If selected correctly, no additional
+allocations will occur during the pipeline execution. One way to find the ideal value is to
+do a complete run over the dataset with the ``memory_stats`` argument set to True and then copy
+the largest allocation value that is printed in the statistics.)code",
       8*1024*1024)  // based on ImageNet heuristics (8MB)
   .AddOptionalArg("affine",
-      R"code(**`mixed` backend only** If internal threads should be affined to CPU cores)code",
+      R"code(Applies only to them **mixed backend type**.
+
+If set to True, the threads from the internal thread pool will be affined to
+the CPU cores Otherwise, the threads are free to be assigned to any CPU core by the OS.)code",
```

Comment:
```suggestion
If set to True, each thread in the internal thread pool will be tied to a specific CPU core.
Otherwise, the threads can be reassigned to any CPU core by the operating system.)code",
```

Reply:
Done